examples/basic/webpack.prod.config.js

```diff
@@ -20,13 +26,26 @@ module.exports = {
 
   plugins: [
     new ExtractTextPlugin('app.css', { allChunks: true }),
+    new webpack.ProvidePlugin({
+      "window.Tether": "tether"
+    }),
   ],
 
   module: {
     loaders: [
       { test: /\.css$/, loader: ExtractTextPlugin.extract('style', 'css!postcss') },
       { test: /\.scss$/, loader: ExtractTextPlugin.extract('style', 'css!postcss!sass') },
-      { test: /\.(woff2?|ttf|eot|svg)$/, loader: 'url?limit=10000' },
+
+      {
+        test: /\.woff2?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+        // Limiting the size of the woff fonts breaks font-awesome ONLY for the extract text plugin
+        // loader: "url?limit=10000"
+        loader: "url"
```

Comment:
@alexfedoseev @wmadden We need to figure this one out. We should be able to have the url limit when doing a prod (extract-text-plugin) deployment.

* http://stackoverflow.com/questions/34133808/webpack-ots-parsing-error-loading-fonts
* http://stackoverflow.com/a/34133809/1009332
* http://stackoverflow.com/a/30763977/1009332